.github/ISSUE_TEMPLATE/contribute.md

```diff
@@ -1,36 +1,21 @@
 ---
 name: Contribute
-about: Let us know you have an example you'd like to contribute
+about: Let us know you have an example you'd like to contribute.
 
 ---
 
-**Description**
-
-A clear and concise description of the concepts in your project and the problem 
-your code solves. Please include any references you used. 
-
-**How to run the example**
-
-A short description of how to run your code, including an example. 
-
 **Link to your repo**
 
-Include a link to the repo you want to contribute to d-wave examples.
-
-https://github.com/dwave-examples
-
-D-Wave will fork your repo. We may make a pull request to your repo if we would 
-like your approval for certain code changes.
+Include a link to the repo you want to contribute to dwave-examples.
 
 **Environment**
- - Python version: [e.g., 3.7.0]
- - Ocean version: [e.g., 2.0]
+ - Operating system
+ - Python version: [e.g., 3.9.0]
+ - Ocean version: [e.g., 3.3.0]
```

Comment:
Yes, that's true. I think it doesn't hurt to have them include this information in their issue anyway, but I would be ok with removing this if you think it's overly redundant.

Reply:
I'm thinking (1) the less contributors have to fill in information, the more likely they'll put the effort into filling in necessary info (2) given two places to fill in a version, half the time the two will show different versions (3) it might confuse some into thinking they should pin a version.

Reply:
That's fair. Will remove.